lib/matplotlib/collections.py

```diff
@@ -98,10 +98,12 @@ def __init__(self, *,
         """
         Parameters
         ----------
-        edgecolors : :mpltype:`color` or list of colors, default: :rc:`patch.edgecolor`
+        edgecolors : :mpltype:`color` or list of colors, optional
             Edge color for each patch making up the collection. The special
             value 'face' can be passed to make the edgecolor match the
-            facecolor.
+            facecolor. If not provided, the value is :rc:`patch.edgecolor` if
+            :rc:`patch.force_edgecolor` is True or *facecolors* is 'none',
+            otherwise 'none'.
```

Comment:
Is the following more understandable? While your description is technically correct, it's hard to understand why these special rule exists. I've
- moved the common/default first 'none' to the beginning
- stated the exception of the default
- without spending extra words hopefully allude to 'none' being not a good value if facecolors is also 'none' because both would be invisible

```suggestion
            facecolor.
            
            If not provided, the value is 'none', i.e. the edge is invisible, unless
            *facecolors* is 'none' or :rc:`patch.force_edgecolor` is True, which both
            result in a fallback to rc:`patch.edgecolor`.
```